bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/package.scala

```diff
@@ -18,7 +18,7 @@ package object immutable {
     // Trim lines, join adjacent lines if non-empty
     def unwrapLines: String = s.lines.map(_.trim).dropWhile(_.isEmpty).map {
       case s0 if s0.isEmpty => "\n"
-      case s0 => s0 + " "
+      case s0               => s0 + " "
```

Comment:
I guess not part of this change but it seems like we could just match the empty string directly:

```scala
case "" => "\n"
```